security/subsystem/src/main/java/org/jboss/as/security/SecurityExtension.java

```diff
@@ -152,18 +155,35 @@ public void initializeParsers(ExtensionParsingContext context) {
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, Namespace.SECURITY_1_0.getUriString(), PARSER);
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, Namespace.SECURITY_1_1.getUriString(), PARSER);
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, Namespace.SECURITY_1_2.getUriString(), PARSER);
+        context.setSubsystemXmlMapping(SUBSYSTEM_NAME, Namespace.SECURITY_1_3.getUriString(), PARSER);
     }
 
     private void registerTransformers(SubsystemRegistration subsystemRegistration) {
         registerTransformers_1_1_0(subsystemRegistration);
         registerTransformers_1_2_0(subsystemRegistration);
+        registerTransformers_1_3_0(subsystemRegistration);
     }
 
     private void registerTransformers_1_1_0(SubsystemRegistration subsystemRegistration) {
         final ResourceTransformationDescriptionBuilder builder = TransformationDescriptionBuilder.Factory.createSubsystemInstance();
         builder.getAttributeBuilder().addRejectCheck(RejectAttributeChecker.SIMPLE_EXPRESSIONS, SecuritySubsystemRootResourceDefinition.DEEP_COPY_SUBJECT_MODE);
         final ResourceTransformationDescriptionBuilder securityDomain = builder.addChildResource(SECURITY_DOMAIN_PATH);
-        securityDomain.getAttributeBuilder().addRejectCheck(RejectAttributeChecker.SIMPLE_EXPRESSIONS, SecurityDomainResourceDefinition.CACHE_TYPE).end();
+        securityDomain.getAttributeBuilder().setDiscard(new DiscardAttributeChecker.DiscardAttributeValueChecker(false, true, new ModelNode(false)),
+                            SecurityDomainResourceDefinition.EXPORT_ELYTRON_REALM)
+                .addRejectCheck(RejectAttributeChecker.SIMPLE_EXPRESSIONS, SecurityDomainResourceDefinition.CACHE_TYPE)
+                .addRejectCheck(new RejectAttributeChecker.DefaultRejectAttributeChecker() {
+
+                    @Override
+                    public String getRejectionLogMessage(final Map<String, ModelNode> attributes) {
+                        return SecurityLogger.ROOT_LOGGER.rejectedAttributeMustUndefinedOrFalse();
+                    }
+
+                    @Override
+                    protected boolean rejectAttribute(final PathAddress address, final String attributeName, final ModelNode attributeValue, final TransformationContext context) {
+                        return attributeValue.asBoolean();
```

Comment:
Actually, you have the discard above which will discard 'false', so you can reject always. This looks like the scenario discussed here:

https://docs.jboss.org/author/display/WFLY10/Domain+Mode+Subsystem+Transformers#DomainModeSubsystemTransformers-Defaultvalueoftheattributeisthesameaslegacyimpliedbehaviour